packages/node/src/Session.spec.ts

```diff
@@ -350,3 +363,87 @@ describe("Session", () => {
     });
   });
 });
+
+describe("getStoredSession", () => {
+  it("returns a logged in Session if a refresh token is available in storage", async () => {
+    const clientAuthentication = mockClientAuthentication();
+    clientAuthentication.getSessionInfo = jest.fn().mockResolvedValue({
+      webId: "https://my.webid",
+      isLoggedIn: true,
+      refreshToken: "some token",
+      issuer: "https://my.idp",
+      sessionId: "mySession",
+    });
+    clientAuthentication.login = jest.fn().mockResolvedValue({
+      webId: "https://my.webid",
+      isLoggedIn: true,
+      sessionId: "mySession",
+    });
+    const dependencies = jest.requireMock("./dependencies");
+    dependencies.getClientAuthenticationWithDependencies = jest
+      .fn()
+      .mockReturnValue(clientAuthentication);
+    const mySession = await getStoredSession("mySession", mockStorage({}));
+    expect(mySession?.info).toEqual(
+      expect.objectContaining({
+        webId: "https://my.webid",
+        isLoggedIn: true,
+        sessionId: "mySession",
+      })
+    );
```

Comment:
Are any other properties on this object valid? Or might it be better to check for the exact object, to ensure that e.g. the refresh token and issuer are not added to it?

```suggestion
    expect(mySession?.info).toStrictEqual(
      {
        webId: "https://my.webid",
        isLoggedIn: true,
        sessionId: "mySession",
      }
    );
```

(And likewise for the other instances of `expect.objectContaining`.)

Reply:
You are right, here we not only check that the values we want are included, but also that the ones we don't aren't present. Updated in ee3c9b9b9c2c65d263f05f07c6dabfc03a45875a.